HAL9666/AuctionMasterBot.py

```diff
@@ -560,4 +561,5 @@ async def clearchannel(ctx):
 
 
 #keep_alive()
-bot.run(os.getenv('DISCORD_TOKEN'))
+if __name__ == "__main__":
+  bot.run(os.getenv('DISCORD_TOKEN'))
```

Comment:
it would make sense to move the intents stuff and bot initialization (lines 86-93) down here too